js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -714,9 +714,7 @@ const EditingController = modules.ViewController.inherit((function() {
         },
 
         _createInsertInfo: function(rowIndex, change) {
-            const insertInfo = {
-                parentKey: change.parentKey
-            };
+            const insertInfo = {};
```

Comment:
It seems that **rowIndex** and **change** arguments are redundant in **_createInsertInfo**.

Reply:
+1

Reply:
done